task_6/task_5/Task1.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1 style="text-align: center;">Добыча драгоценных камней</h1>
+    <table border="1" style="border-collapse: collapse; width: 100%;">
+<tr style="background-color: green; text-align: center;">
+    <td></td>
+    <td><strong>2010</strong> </td>
+    <td><strong>2012</strong> </td>
+    <td><strong>2014</strong> </td>
+</tr>
+<tr style="background-color: #ffffd1;">
+    <td style="width: 100px; height: 30px; padding: 10px;">Сапфиры</td>
```

Comment:
heigth is redundant in this case. Remove it

Reply:
Done